### DISCUSSION TITLE
Doc to explain networking model

### DISCUSSION THREAD

**Discussion by r2an9** [vhive-serverless/vHive#288] [Q&A] [answered]
Is there any document describing the networking model?
At a glance of the code, the same IP address is assigned to both firecracker VM and the queue-proxy container. How is it suppose to work? 
If one function needs to get something from network, will it all work out?

**Comment by ustiugov** [comment, marked as answer]
hi @xiregele, thank you for your interest in vHive. 

To understand the function invocation part of networking, you may refer to the diagram from the main README:
![image](https://user-images.githubusercontent.com/15651770/122755439-e9851700-d28c-11eb-8249-43275e835920.png)

As for user code inside functions, it can freely access any resources in the same k8s cluster or in the internet. However, note that vHive doesn't support inbound connections to function instances. If you need to expose function instances to the outside world, you need to configure the iptables separately.

It would be great if you could briefly introduce yourself (e.g., mention which organization or university do you work for) and describe your usage scenario so that we could provide more specific support to you. Feel free to DM me on the Firecracker Slack if you prefer communicating there.
